event-log/README.md

```diff
@@ -0,0 +1,243 @@
+# event-log
+
+This is a microservice which provides CRUD operations for Event Log DB.
+
+## API
+
+| Method | Path                                     | Description                                                    |
+|--------|------------------------------------------|----------------------------------------------------------------|
+|  POST  | ```/events```                            | Creates an event with a `NEW` status                           |
+|  PATCH | ```/events```                            | Changes events' data by applying the given patch               |
+|  GET   | ```/events/latest```                     | Finds events for all the projects with the latest `event_date` |
+|  PATCH | ```/events/:id/projects/:id/status```    | Updates event status                                           |
```

Comment:
Do I understand that each event is uniquely linked to one project? If so, I think it can simply be a `PATCH` to `/events/:id/`.

Reply:
That would be nice but unfortunately, an event is uniquely identified by a compound key `event id -> project id` as potentially there could be two events with the same `event id` belonging to two different projects.

Reply:
Ah, I see. Is it then fair to say that the event Id could be considered an event type? I think the accepted way to handle composite ids in REST would be 
`/events/:event-type/:project-id`